pulpcore/openapi/__init__.py

```diff
@@ -389,7 +390,7 @@ def parse(self, request, public):
                 continue
 
             # Removes html tags from OpenAPI schema
-            if "include_html" not in request.query_params:
+            if request and "include_html" not in request.query_params:
```

Comment:
```suggestion
            if request is None or "include_html" not in request.query_params:
```
As `include_html` is to be requested specifically, we should default to `strip_tags`.